kclvm/sema/src/advanced_resolver/mod.rs

```diff
@@ -396,47 +396,12 @@ mod tests {
                 ],
             ),
             (
-                "src/advanced_resolver/test_data/pkg/pkg.k"
+                "src/advanced_resolver/test_data/import_test/d.k"
```

Comment:
Considering improve code according to the [CodiumAI-Agent](https://github.com/CodiumAI-Agent) comments and fix CI unit tests